app/src/viewmodels/home/HomeScreenViewModel.tsx

```diff
@@ -83,13 +106,14 @@ export function useHomeScreenViewModel(
 
   return {
     userIsGuest,
-    challenges,
+    challenges: blurredChallenges,
```

Comment:
What did you want to do there ? It seems it's not used in the View File and it should probably not be there.

Reply:
Yes sorry that was a try to blur the challenge from a certain point but since the library was not safe for android I abandoned the idea and forgot to remove it. Good catch !